otpad/otpad.py

```diff
@@ -1,20 +1,68 @@
 from operator import xor
+import hmac
+import hashlib
+import base64
 
 
-def onetimepad(key1, key2):
+def pad(key, key_to_encrypt, hmac_key=None):
```

Comment:
consider naming these variables `plaintext`, `key` and `hmac_secret` - should just make things a little clearer.